### PR TITLE
Menu suggestion

### LINKED ISSUE #10: Add Configuration Menu and View

### PR DESCRIPTION
**Pull request by mehlium**
Tries to implement wallet navigation as talked about in #10. Also adds basic "back" and "forward" navigation as talked about in #4. Still a very basic UI for this. Went a bit overboard on this and implemented balance API and QR code as well (had it available in my previous kaios-wallet thingy).
 
Copied send/receive views into the wallet-views folder. If we move forward with this we can delete those on the "top level" and add a menu→send flow.

nano-kaois-1

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -66,6 +67,7 @@ export default {
 			sourceMap: !production,
 			inlineSources: !production
 		}),
+		nodePolyfills(),
```
Comment (mehlium) [inline rollup.config.js:70]: Necessary to get the qrcode dependency working. :thinking:

File: src/machinery/models.ts [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-export interface Account {
+export interface NanoAccount {
     alias: string
     address: NanoAddress
 }
 
-export interface Wallet {
-    accounts: Account[]
+export interface NanoWallet {
+    accounts: NanoAccount[]
```
Comment (mehlium) [inline src/machinery/models.ts:10]: Wallet/Account became a bit generic and name clashed with the views.

File: src/view/Wallet.svelte [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+            {
+                alias: "Main",
+                address: "nano_3niceeeyiaa86k58zhaeygxfkuzgffjtwju9ep33z9c8qekmr3iuc95jbqc8"
+            },
+            {
+                alias: "Savings",
+                address: "nano_1hzoje373eapce4ses7xsx539suww5555hi9q8i8j7hpbayzxq4c4nn91hr8"
+            }
+        ],
+        seed: "some-seed"
```
Comment (Citrullin) [inline src/view/Wallet.svelte:21]: Well done! No seed in it :D

**Comment by Citrullin** [comment]
Okay, it doesn't run on my phone. I will debug it.
/e I checked. I started to not work after bdb4e2da70707e32a650df2f2bb9a69844c7d88d
My fault, I didn't checked it on the phone the whole time. I should have.

**Comment by Citrullin** [comment]
Okay, I found the error:
```12:58:30.188 SyntaxError: missing = in const declaration1bundle.js:1:315```

This is the issue: ```for(const n in e)```. Probably not supported by the Firefox version, so we need a polyfill for that. 
It is a bug in Firefox: https://bugzilla.mozilla.org/show_bug.cgi?id=1101653

We should use Babel, I guess.

**Comment by mehlium** [comment]
I'm merging this then, and made a new issue for the problem: https://github.com/KaiOS-Nano-wallet/nano-kaios/issues/21

**Pull request merged**